cookbooks/aws-parallelcluster-environment/spec/unit/recipes/config_default_user_home_spec.rb

```diff
@@ -22,7 +22,31 @@
           expect(chef_run.node['cluster']['cluster_user_home']).to eq('/local/home/user')
           is_expected.to start_service("sshd")
         end
+
+        it 'moves the cluster user home directory with data integrity check' do
+          user_home = "/home/user"
+          user_local_home = "/local/home/user"
```

Comment:
*Minor on CodeStyle / Knowledge Sharing*
It's a good practice in testing code to use values conveying the assumptions you have on such variables.
This is a good practice to leverage test code as a complementary form of documentation (not an absolute one of corse).
https://softwareengineering.stackexchange.com/questions/154615/are-unit-tests-really-used-as-documentation

Example:
`user_local_home` can be whatever absolute path in the system. 
If you set it to `/local/home/user` it  seems that there is a requirement for that directory to be within `/local`, since it is a notable folder in UNIX, but this is not the case.
If you set it to `/a/local/user/home` or `/whatever/local/user/home` or `/WHATEVER_DIR`, you convey the right assumptions: it can be whatever absolute path.

Reply:
Thank you for your sharing. Have corrected it to `"/custom/local/home/user"`

Reply:
I see, let's keep it as it is as it's not worth it to change parts of the code that are out of the PR scope.